### DISCUSSION TITLE
Use external API:s as a Collection

### DISCUSSION THREAD

**Discussion by n1mda** [directus/directus#17549] [Archive]
Hello, I am wondering if it is possible to create a Collection in Directus that fetches and creates data from a remote API?
I have an existing remote API that allows me to list paginated items, create new items and edit them etc.
I would like to use Directus to view and manage these items, instead of creating a custom UI for the API. Is there any documentation on how to achieve this?

**Comment by casualmatt** [comment]
I just post some days a go a full Feature Requests, with an example on how to integrate external data in directus, maybe you have feedback to improve my ideas. 

https://github.com/directus/directus/discussions/17504
